server/tserver/src/main/java/org/apache/accumulo/tserver/log/TabletServerLogger.java

```diff
@@ -338,10 +338,9 @@ private void write(final Collection<CommitSession> sessions, boolean mincFinish,
           success = (currentLogSet == logSetId.get());
         }
       } catch (DfsLogger.LogClosedException ex) {
-        writeRetry.logRetry(log, "Logs closed while writing");
+        writeRetry.logRetry(log, "Logs closed while writing", ex);
```

Comment:
Seems like the warning will still happen during normal operations?

Reply:
I don't have a good feel for how many times the code retries in this situation.  I wonder if the warning will show up when there is not a problem.

Reply:
I bet your continuous ingest will tell you.

Reply:
Now that the first message is debug, I can guarantee this is no worse that what we had before in terms of logging.

Reply:
> I bet your continuous ingest will tell you.

yeah, it will.  I think the fix looks good.  Testing will tell for sure.

Reply:
In the current test I am running, seeing the following on the monitor.  Noticed there was one time where the retry attempt was 3.

![accumulo-4832](https://user-images.githubusercontent.com/1268739/36695448-63943bf4-1b0f-11e8-958b-23465f5a96a0.png)

Reply:
These were the final counts from the test I was running against 1.7.4 RC0.  

![accumulo-4832-2](https://user-images.githubusercontent.com/1268739/36748112-e63f63c0-1bc4-11e8-9e27-36c91bc48923.png)